docs/HowTo/GetStarted/monitoring.md

```diff
@@ -0,0 +1,46 @@
+# Monitoring Quorum Node
+
+GoQouroum version `2.6.0` upgraded `geth` version to 1.9.7. One of the key features the upgrade brought in, was the ability to collect enhanced metrics which can then be easily exposed to outside world and integrated with different visualization tools for node monitoring. Please refer to [Geth v1.9.0](https://blog.ethereum.org/2019/07/10/geth-v1-9-0/) for complete list of features added in `geth` version 1.9.0.
+
+This page summarizes ways in which a GoQuorum node can be brought up to collect enhanced metrics and thus enable node monitoring.
+
+## Metrics Collection
+
+For node monitoring, the first step is to gather the metrics data and expose the gathered data for consumption by the monitoring tools. GoQuroum node can be instructed to gather metrics data and expose the same to external world via [ExpVars](https://golang.org/pkg/expvar/), [InfluxDB](https://www.influxdata.com/products/influxdb-overview/) or [Prometheus](https://prometheus.io/)
+
+* Bringing up GoQuorum node with `--metrics --pprof` will expose the metrics data in `expvar` format at `http://127.0.0.1:6060/debug/metrics`
+
+* Bringing up GoQuorum node with `--metrics --pprof` will also expose the metrics data in Prometheus format at `http://127.0.0.1:6060/debug/metrics/prometheus`
```

Comment:
only one sentence for both with the two endpoints would be clearer:

```
Use GoQuorum `--metrics` and  `--pprof` [command line options](#command-line-options) to expose the metrics data:

* In ExpVar format at `http://127.0.0.1:6060/debug/metrics`
* In Prometheus format at `http://127.0.0.1:6060/debug/metrics/prometheus`
```

Reply:
Thanks @NicolasMassart  for all the suggestions. I have updated the docs to reflect the same.